### PR TITLE
SOE-623: Updated tax weight for grouping views

### PR DESCRIPTION
**Pull request by cjwest**
This adds and updates the sort by taxonomy weight for most of the views.

### CODE REVIEW

**Comment by jbickar** [comment]
Can you describe steps to test this pull request? What should I see (and where) with the current `7.x-5.x-dev` branch, vs. what should I see (and where) on the `SOE-623` branch? The JIRA ticket does not have much detail.

**Comment by cjwest** [comment]
Hi @jbickar,

It appears that Greg is having trouble getting Behat tests working. If you still want to look at this, I'd really appreciate it.

Here are the steps to testing this:
1) Look at each layout views for person. 
2) make sure each works as expected. 
3) add an extra staff member and student and give them tags and verify that they show up in the order as expected, particularly in the grouped views.

If you use the @cjwest in the comments, I will respond much more quickly!

**Comment by jbickar** [comment]
@cjwest can you give me more details behind "each layout views for person" (e.g., what paths, specifically), and "works as expected" (what should they look like?)?

**Comment by cjwest** [comment]
Hi @jbickar,
Thanks for looking into this!

I updated the [ticket](https://stanfordits.atlassian.net/browse/SOE-623?focusedCommentId=42231&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-42231) with more specific info.

**Comment by cjwest** [comment]
Hey @cynmij,
Here's the PR for person. Let me know if you have any questions.
Thanks for looking at it!!!

CJ

**Comment by cynmij** [comment]
views all work including rearranging taxonomy terms

**Comment by sherakama** [comment]
We have a LOT of people views installed by default now... I think it is time to clean this up.

ex:
screen shot 2016-03-29 at 12 05 16 pm

File: modules/stanford_person_views/stanford_person_views.views_default.inc [hunk 1]
```diff
@@ -324,6 +324,141 @@ function stanford_person_views_views_default_views() {
...
+  $handler->display->display_options['sorts']['field_s_person_last_name_value']['table'] = 'field_data_field_s_person_last_name';
+  $handler->display->display_options['sorts']['field_s_person_last_name_value']['field'] = 'field_s_person_last_name_value';
+  /* Sort criterion: Content: First name (field_s_person_first_name) */
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['id'] = 'field_s_person_first_name_value';
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['table'] = 'field_data_field_s_person_first_name';
+  $handler->display->display_options['sorts']['field_s_person_first_name_value']['field'] = 'field_s_person_first_name_value';
+  $handler->display->display_options['path'] = 'people/%/cap-list/grouped/affiliation';
+
+  /* Display: Group by Student Page */
+  $handler = $view->new_display('page', 'Group by Student Page', 'page_2');
```
Comment (sherakama) [inline modules/stanford_person_views/stanford_person_views.views_default.inc]: The title for this view is Student but this view exists at `people/%/cap-list/grouped`

Comment (sherakama) [inline modules/stanford_person_views/stanford_person_views.views_default.inc]: Please give this view a better machine name.

**Comment by sherakama** [comment]
I left a couple of label comments but otherwise this is good. 

I have added a ticket to the backlog to look at organizing all of the views. It is becoming overwhelming to know or to understand when to use one over the other. I won't let the number of views prevent this merge, however. Please update the labels and then it looks good to me.

File: modules/stanford_person_views/stanford_person_views.views_default.inc (hunk 1 above)
Comment (cjwest) [inline modules/stanford_person_views/stanford_person_views.views_default.inc]: I'm not sure what action you're suggesting here:
"The title for this view is Student but this view exists at people/%/cap-list/grouped"

**Comment by cjwest** [comment]
@sherakama: please review.

**Comment by cjwest** [comment]
Woot!

**Pull request merged**